src/crud/TinyDelete.java

```diff
@@ -51,97 +56,139 @@ protected void processRequest(HttpServletRequest request, HttpServletResponse re
                     sb.append(line);
                 }
                 reader.close();
                 for (Map.Entry<String, List<String>> entries : connection.getHeaderFields().entrySet()) {
                     String values = "";
                     String removeBraks = entries.getValue().toString();
-                    values = removeBraks.substring(1, removeBraks.length() -1);
-                    if(null != entries.getKey() && !entries.getKey().equals("Transfer-Encoding")){
+                    values = removeBraks.substring(1, removeBraks.length() - 1);
+                    if (null != entries.getKey() && !entries.getKey().equals("Transfer-Encoding")) {
                         response.setHeader(entries.getKey(), values);
                     }
                 }
-            }
-            catch(IOException ex){
-                //Need to get the response RERUM sent back.
-                BufferedReader error = new BufferedReader(new InputStreamReader(connection.getErrorStream(),"utf-8"));
+            } catch (IOException ex) {
+                BufferedReader error = new BufferedReader(new InputStreamReader(connection.getErrorStream(), "utf-8"));
                 String errorLine = "";
-                while ((errorLine = error.readLine()) != null){  
+                while ((errorLine = error.readLine()) != null) {
                     sb.append(errorLine);
-                } 
+                }
                 error.close();
             }
             connection.disconnect();
         }
-        if(manager.getAPISetting().equals("true")){
-            response.setHeader("Access-Control-Allow-Origin", "*"); //To use this as an API, it must contain CORS headers
-            response.setHeader("Access-Control-Expose-Headers", "*"); //Headresponse.setHeader("Access-Control-Allow-Methods", "DELETE");ers are restricted, unless you explicitly expose them.  Darn Browsers.
+
+        if (manager.getAPISetting().equals("true")) {
+            response.setHeader("Access-Control-Allow-Origin", "*");
+            response.setHeader("Access-Control-Expose-Headers", "*");
             response.setHeader("Access-Control-Allow-Headers", "*");
             response.setHeader("Access-Control-Allow-Methods", "DELETE");
         }
-        
+
         response.setStatus(codeOverwrite);
         response.setHeader("Content-Type", "text/plain; charset=utf-8");
         response.setCharacterEncoding("UTF-8");
         response.getWriter().print(sb.toString());
     }
 
+    /**
+     * Helper method to validate the Bearer token.
+     * 
+     * @param token the Bearer token to validate
+     * @return true if the token is valid, false otherwise
+     */
+    private boolean validateBearerToken(String token) {
+        try {
+            FirebaseToken decodedToken = FirebaseAuth.getInstance().verifyIdToken(token);
+            // Token is valid
+            return true;
+        } catch (Exception e) {
+            // Invalid token
```

Comment:
Since you are just returning `false` here, we never see what Exception this is.  You'll want to know what specific Exception is happening here, since it has to do with the firebase you want to access.